packages/components/src/box-control/icon.js

```diff
@@ -29,10 +29,10 @@ export default function BoxControlIcon( {
 		return side === 'all' || side === value;
 	};
 
-	const top = getSide( 'top' );
-	const right = getSide( 'right' );
-	const bottom = getSide( 'bottom' );
-	const left = getSide( 'left' );
+	const top = getSide( 'top' ) || getSide( 'vertical' );
+	const right = getSide( 'right' ) || getSide( 'horizontal' );
+	const bottom = getSide( 'bottom' ) || getSide( 'vertical' );
+	const left = getSide( 'left' ) || getSide( 'horizontal' );
```

Comment:
Not sure if it applies, but in case the `getSize()` function can return `0`, then using the `||` could lead to some unintended behaviour  — maybe better to use `??` ?

Reply:
Good point about falsy values. In this case `getSide` always returns a boolean, and its value is used in a truthy way in [icon styles](https://github.com/andrewserong/gutenberg/blob/ecbcae6a4578346b63d7b7985b493c7c1ca6d50a/packages/components/src/box-control/styles/box-control-icon-styles.js#L27), so it's resilient to `0` and other falsy values. Whereas switching to `??` would break when the left-hand operand returns false as the right-hand would never get evaluated. So, in this case I think `||` is a better fit, but we could always wrap the whole thing in a double negation (`!!`) if you think it's risky as is?

Reply:
Makes sense! I just got confused by the name of the function — I didn't expect it to return a `boolean` (maybe something like `hasSize` would be more clear?)

But regardless, if that function always returns a `boolean`, then there's no need to change the implementation IMO

Reply:
> I didn't expect it to return a boolean (maybe something like hasSize would be more clear?)

Good thinking, that improves readability 👍. I've just updated the function name.